hyperv-ubuntu-160.04-cloud-init.json

```diff
@@ -0,0 +1,65 @@
+{
+  "variables": {
+    "vm_name": "ubuntu-xenial",
+    "cpu": "2",
+    "ram_size": "2048",
+    "disk_size": "3072",
+    "iso_url": "http://releases.ubuntu.com/16.04/ubuntu-16.04.2-server-amd64.iso",
+    "iso_checksum_type": "sha1",
+    "iso_checksum": "F529548FA7468F2D8413B8427D8E383B830DF5F6",
+    "hyperv_switchname": "{{env `hyperv_switchname`}}",
+	"username" : "ubuntu",
+	"password" : "ubuntu"
+  },
+  "builders": [
+  {
+    "vm_name":"{{user `vm_name`}}",
+    "type": "hyperv-iso",
+    "disk_size": "{{user `disk_size`}}",
+    "guest_additions_mode":"disable",
+    "iso_url": "{{user `iso_url`}}",
+    "iso_checksum_type": "{{user `iso_checksum_type`}}",
+    "iso_checksum": "{{user `iso_checksum`}}",
+    "communicator":"ssh",
+    "ssh_username": "{{user `username`}}",
+    "ssh_password": "{{user `password`}}",
+    "ssh_timeout" : "4h",
+    "http_directory": "./linux/ubuntu/http/",
+    "boot_wait": "5s",
+    "boot_command": [
+      "<esc><wait10><esc><esc><enter><wait>",
+      "set gfxpayload=1024x768<enter>",
+      "linux /install/vmlinuz ",
+      "preseed/url=http://{{.HTTPIP}}:{{.HTTPPort}}/preseed.cfg ",
```

Comment:
I justed noticed that I have not changed the preseed.cfg URL to 

`"preseed/url=http://{{.HTTPIP}}:{{.HTTPPort}}/preseed-cloud-init.cfg ", `

This is fixed allready in my fork:
https://github.com/fwwasol/packer-baseboxes/commit/74e75d0d90ede85e68f25ac79808c54ec2ec7f80